lib/screens/v2/candidate_generator/dup/departures.ex

```diff
@@ -0,0 +1,252 @@
+defmodule Screens.V2.CandidateGenerator.Dup.Departures do
+  @moduledoc false
+
+  alias Screens.Alerts.Alert
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Departures
+  alias Screens.Config.V2.Departures.{Headway, Query, Section}
+  alias Screens.Config.V2.Departures.Query.Params
+  alias Screens.Config.V2.Dup
+  alias Screens.SignsUiConfig
+  alias Screens.Util
+  alias Screens.V2.CandidateGenerator.Widgets
+  alias Screens.V2.WidgetInstance.Departures, as: DeparturesWidget
+  alias Screens.V2.WidgetInstance.DeparturesNoData
+
+  @branch_stations ["place-kencl", "place-jfk", "place-coecl"]
+  @branch_terminals [
+    "Boston College",
+    "Cleveland Circle",
+    "Riverside",
+    "Heath Street",
+    "Ashmont",
+    "Braintree"
+  ]
```

Comment:
Currently checking with PaulH to see if this logic is necessary in v2. This was pulled straight from v1, but is not in the spec and am unsure what the initial reasoning was for why it exists.

Reply:
Had a chat with Paul and this logic does need to stay. Pasting his explanation:

```
Say there's a JFK/UMass - Quincy Center shuttle.

We wouldn't want to be in headway mode at JFK/UMass because trains are running Alewife - Ashmont normally (there'd be predictions like normal) (edited) 

So that's why "make sure that the stop isn't a branch stop" makes sense.

On the other hand, if there's a Broadway - JFK/UMass shuttle then we would show headways at JFK/UMass, because we're running trains from there to Ashmont and to Braintree, but we won't have predictions

So that's why "headsign is not a branch terminal"
```